### PR TITLE
Put dataset permissions in sharing tab

### LINKED ISSUE #4683: Dataset Permission should be in separate tab
With the recent permissions system refactoring, the `allowed teams` option should be more prominent since it is important property of the dataset.

I suggest to move it out of the `Metadata` tab into its own, separate tab `Permissions` (or `Access Permissions`) to make this feature more prominent. This is a cosmetic change only. No further functionality needs to be changed. 

image

### PR DESCRIPTION
**Pull request by grittaweisheit**
This PR moves a dataset's permission settings to the Sharing & Permissions tab.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to the editing view of a dataset
- there should be a "Sharing & Permissions" tab 
- there should be the input field for teams that are allowed to access the dataset
- in the "Metadata" tab should be no such input field

### Issues:
- fixes #4683

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] ~~Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable~~
- [ ] ~~Updated [documentation](../blob/master/docs) if applicable~~
- [ ] ~~Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- [ ] ~~Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
Very cool! However, I noticed that the "warning" icon which appears when one doesn't have any "allowed teams" assigned is shown in the wrong tab title. See screenshot:

image

Could you please move that icon to the sharing & permission tab? Other than that, the PR looks fine to me!

**Comment by grittaweisheit** [comment]
Sure! I totally didn't think about this icon. It should now be in the right position.

**Comment by philippotto** [comment]
(btw, you don't need to assign the `good first issue` label to the PR, as we typically use that only for filtering the issues and not the PRs)

File: CHANGELOG.unreleased.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 - Disabled the autofill feature of the brush when using this tool to erase data. [#4729](https://github.com/scalableminds/webknossos/pull/4729)
 - The rotation buttons of the 3D-viewport no longer change the zoom. [#4750](https://github.com/scalableminds/webknossos/pull/4750)
 - Improved the performance of applying agglomerate files. [#4706](https://github.com/scalableminds/webknossos/pull/4706)
+- Moved a dataset's permission settings to the sharing tab. [#4683](https://github.com/scalableminds/webknossos/pull/4763)
```
Comment (philippotto) [inline CHANGELOG.unreleased.md:24]: Sorry, I'm a bit nitpicking:

```suggestion
- In the Edit/Import Dataset form, the "Sharing" tab was renamed to "Sharing & Permissions". Also, existing permission-related settings were moved to that tab.  [#4683](https://github.com/scalableminds/webknossos/pull/4763)
```

Comment (grittaweisheit) [reply CHANGELOG.unreleased.md:24]: Fair point. It is a lot more specific! Thanks for the suggestion :)

**Pull request merged**